cmake/DetectCodeCoverage.cmake

```diff
@@ -20,7 +26,8 @@ if ( GCOV AND
      LCOV AND
```

Comment:
> I use Travis with [codecov](codecov.io), which just a parses all the gcov output files.

Cool! Will codecov.io work with Charm++ stuff and can it filter out unwanted files, such as https://github.com/quinoacomputing/quinoa/blob/master/cmake/CodeCoverage.cmake#L80 does?